src/contracts/Repository/SearchService.php

```diff
@@ -45,19 +49,24 @@ interface SearchService
     public const CAPABILITY_FACETS = 2;
 
     /**
-     * Capability flag for custom fields feature for use with {@see ::supports()}.
+     * Capability flag for custom fields feature.
+     *
+     * To use with {@see SearchService::supports()}.
      *
-     * Custom fields is the capability for search engines to 1. allow you to extend the search index via plugins to
-     * generate custom fields, like a different representation (format, ...) of an existing field or similar. And 2.
-     * allow you on some search criteria to specify this custom field to rather query on that instead of the default
-     * field generated by the system.
+     * Custom fields is the capability for search engines to:
+     * - Allow you to extend the search index via plugins to
```

Comment:
```suggestion
     * - allow you to extend the search index via plugins to
```

Reply:
> ```suggestion
>      * - allow you to extend the search index via plugins to
> ```
Doesn't that depend on how we finish the sentence in bullet point? Meaning, if it ends with:

- Dot = capital letters for bullet points.
- Comma = small letters.

But don't quote me on that, maybe English grammar rules are different?

Reply:
@konradoboza As a list of full sentences, there should be a cap as @alongosz also said.